src/metabase/util/tracking.clj

```diff
@@ -0,0 +1,54 @@
+(ns metabase.util.tracking
+  (:require [clj-http.client :as client]))
+
+(def ^:private ^:const ^String google-analytics-collect-url "http://www.google-analytics.com/collect")
+
+(def ^:private ^Integer anonymous-id
+  "Generate an anonymous id. Don't worry too much about hash collisions or localhost cases, etc.
+   The goal is to be able to get a rough sense for how many different hosts are throwing a specific error/event." 
+  (hash (str (java.net.InetAddress/getLocalHost))))
+
+(defn- google-analytics-tracking-code
+  "Get the GA id for the main metabase application"
+  []
+  (require 'metabase.public-settings)
+  ((resolve 'metabase.public-settings) :ga_code))
```

Comment:
It might make sense actually just to make the ga code a separate definition, e.g. in `public_settings.clj`

```clojure
(def ^String ga-code 
   "write some documentation here"
   "UA-60817802-1")
```

Then to get it you just do

```clojure
(require 'metabase.public-settings)
@(resolve 'metabase.public-settings/ga-code)
```

(`resolve` returns a var (you can think of it like a sort of pointer), and to get the value of a var you need to `deref` it. `@x` is syntactic sugar for `(deref x)`.